data/MapsConfig/maps.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
change the hierarchy for this also to data/maps. Because this file is a resource file not a configuration file.